rtx-remix/mods/gameReadyAssets/MagicUSDA.py

```diff
@@ -0,0 +1,263 @@
+import argparse
```

Comment:
Is there a reason to keep this script around? If yes, would be nice to add a comment on top specifying that this script came from Kim's repository, specifically from:
https://github.com/Kim2091/RTXRemixTools/blob/USDRefactor/MagicUSDA/MagicUSDA.py

Although it seems very different from the actual file in his repo 🤔

Reply:
oops, I was testing stuff for Kim, didn't mean to leave it in here.  Delete.